Stripe/PublicHeaders/STPPaymentMethodsViewController.h

```diff
@@ -75,6 +75,17 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic, strong, nullable)STPUserInformation *prefilledInformation;
 
 /**
+ Determines whether or not the user is able to delete payment methods using this view controller
+
+ The user can tap the "Edit" button or swipe left on a payment method to delete it if enabled.
+
+ Currently, the user is not allowed to delete the selected payment method but this may change in the future.
+
+ Default is NO.
+ */
+@property (nonatomic, assign, readwrite) BOOL canDeletePaymentMethods;
```

Comment:
I think we should move this to `STPPaymentConfiguration` – it's where all our configuration options live right now, and we can avoid redefining it in a bunch of places / forwarding.

I also wonder whether we should default it to on. This seems like a pretty safe feature to introduce for everyone, and since we introduced ephemeral keys pretty recently, it won't affect that many people.

We should also add a note that this feature won't be available if you're using STPBackendAPIAdapter, and maybe point to https://github.com/stripe/stripe-ios/blob/master/MIGRATING.md#migration-from-versions--1100